ENGAGEHF/Bluetooth/Views/ViewElements.swift

```diff
@@ -0,0 +1,36 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziViews
+import SwiftUI
+
+
+struct CloseButtonLayer: View {
+    @Environment(\.dismiss) private var dismiss
+    @Binding private var viewState: ViewState
+    
```

Comment:
I would propose to stay consistent about the spacing between different elements in a type, e.g., stored properties and computed properties. Also applies to other parts of the app; we should have two empty lines between these elements to separate this out:
```suggestion
    
    
```